.github/workflows/rebase-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: Rebase a pr
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  rebase_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@master
+      - name: Rebase PR
+        if: |
+          github.event_name == 'pull_request' &&
+          github.event.action == 'labeled' &&
+          github.event.label.name == 'rebase-pr'
```

Comment:
Just out of curiosity since I'm not too familiar with Github actions (yet :grin:). Do we need to check `event_name` and `event.action` again? (_aren't these covered by the `on:` field?